### DISCUSSION TITLE
Compiling projects using Cmakefile

### DISCUSSION THREAD

**Discussion by prajwalresearch** [coin-or/Ipopt#507] [Q&A] [answered]
Hello everyone , newbie here. I have installed the library using coin brew and and able to successfully compile and run the examples given  using makefile.  I have ubuntu 18.04.
I have a VS code project where  I have compiled the project using Cmake file . I cannot compile the project using makefile as there is some libraries when authors advice to use of Cmake only like libtorch.
So I want to ask if there is a way to compile a project using Cmakefile . ? Sorry if the question seems silly.

**Comment by svigerske** [comment, marked as answer]
There are no plans or resources to maintain additional buildsystems here at coin-or/Ipopt. There may exist some cmake buildsystems for Ipopt made by users out there.
